NuRadioReco/modules/io/RNO_G/readRNOGDataMattak.py

```diff
@@ -775,16 +775,18 @@ def _get_event(self, event_info, waveforms):
         evt = NuRadioReco.framework.event.Event(event_info.run, event_info.eventNumber)
         station = NuRadioReco.framework.station.Station(event_info.station)
         station.set_station_time(astropy.time.Time(trigger_time, format='unix'))
+        evt.set_event_time(astropy.time.Time(trigger_time, format='unix'))  # Typically event time = station time
```

Comment:
Converting to an astropy.time.Time object isn't quite ´free'; it's fast enough that it doesn't matter here but seeing as you sometimes like superfluous optimizations, it could be made quicker here by creating it only once.

Reply:
I removed that line. I think we agree that for data the event time does not need to be set.

Reply:
That's also fine. What I thought was actually `evt.set_event_time(station.get_station_time())` (or an equivalent) so you don't call the 'expensive' `astropy.time.Time` initialization twice. But I have no objections to not setting the event time in data.